api/osf_api.py

```diff
@@ -1,7 +1,8 @@
+import datetime
```

Comment:
Since you changed `from datetime import datetime` to `import datetime`, you need to fix the other calls to datetime in this code (ex. ln. 646)

Reply:
I just double-checked and ln. 646 is the only place it needs to be updated.

Reply:
I changed it to datetime.datetime.now().year
Hope this works

Reply:
@Ramyapatil712 and @felliott , this PR is out of date with `develop`, Ramya already changed the `datetime` import in her last PR, she might need to rebase and fix the conflicts first, otherwise we are reviewing some code that are already in `develop`.